content/_code-samples/escrow/js/create-escrow.js

```diff
@@ -1,57 +1,61 @@
-
-console.log('Condition:', conditionHex)
-console.log('Fulfillment:', myFulfillment.serializeBinary().toString('hex').toUpperCase())
-
-// Construct transaction
-const currentTime = new Date()
-const myEscrow = {
-  "destination": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn", // Destination can be same as source
-  "destinationTag": 2017,
-  "amount": "0.1113", //decimal XRP
-  "condition": conditionHex,
-  "allowExecuteAfter": currentTime.toISOString() // can be executed right away if the condition is met
-}
-const myInstructions = {
-  maxLedgerVersionOffset: 5
-}
-
-// Connect and submit
-const api = new RippleAPI({server: 'wss://s2.ripple.com'})
-
-function submitTransaction(lastClosedLedgerVersion, prepared, secret) {
-  const signedData = api.sign(prepared.txJSON, secret)
-  console.log('Transaction ID: ', signedData.id)
-  return api.submit(signedData.signedTransaction).then(data => {
-    console.log('Tentative Result: ', data.resultCode)
-    console.log('Tentative Message: ', data.resultMessage)
-  })
-}
-
-api.connect().then(() => {
-  console.log('Connected')
-  return api.prepareEscrowCreation(myAddr, myEscrow, myInstructions)
-}).then(prepared => {
-  console.log('EscrowCreation Prepared')
-  return api.getLedger().then(ledger => {
-    console.log('Current Ledger', ledger.ledgerVersion)
-    return submitTransaction(ledger.ledgerVersion, prepared, mySecret)
-  })
-}).then(() => {
-  api.disconnect().then(() => {
-    console.log('api disconnected')
-    process.exit()
-  })
-}).catch(console.error)
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // Construct an EscrowCreate transaction to create an Escrow natively on the XRPL
+  // https://xrpl.org/escrow.html#escrow
+  // https://xrpl.org/escrowcreate.html#escrowcreate
+
+  async function main() {
+    // Connect to a testnet node
+    console.log("Connecting to Testnet...")
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233')
+    await client.connect()
+  
+    // Get account credentials from the Testnet Faucet
+    console.log("Requesting addresses from the Testnet faucet...")
+    const { wallet } = await client.fundWallet()
+    const { wallet: reciepient_wallet } = await client.fundWallet()
+
+    console.log(" Sending Account:", wallet.address)
+    console.log("            Seed:", wallet.seed)
+
+    console.log("\n Receiving Account:", reciepient_wallet.address)
+
+    // Get current datetime
+    const date = new Date();
+
+    // ISO8601 timestmap to a ripple timestamp.
+    // After the FinishAfter time has passed, you are given 20 seconds to finish ('claim') the escrow
+    // After the CancelAfter time has passed, you're unable to finish the Escrow but you're allowed to Cancel it
+    // You could configure CancelAfter and FinishAfter to a different time but the CancelAfter field must always be sooner than the FinishAfter field
```

Comment:
This one is backwards
```suggestion
    // You could configure FinishAfter and CancelAfter to a different time but the FinishAfter field must always be sooner than the CancelAfter field
```